client/src/reducers/loginReducer.js

```diff
@@ -0,0 +1,12 @@
+import { LOGIN } from '../actions/types'
+import { browserHistory } from 'react-router'
+import credentials from '../data/login.json'
+
+export default function (state = false, action) {
+  switch (action.type) {
+    case LOGIN:
+      return (action.payload.email === credentials.email && action.payload.password === credentials.password)
```

Comment:
For this, it may be better to do something like the following:
```js
const defaultState = {
  loggedIn: false
}

export default function (state = defaultState, action) {
  switch (action.type) {
    case LOGIN:
      return {...state, loggedIn: true}
...
```